packages/app-desktop/InteropServiceHelper.test.ts

```
@@ -0,0 +1,66 @@
+
+jest.mock('./services/bridge', () => ({
+	__esModule: true,
+	default: () => ({ newBrowserWindow: () => mockWindow }),
+}));
+
+describe('InteropServiceHelper', () => {
+	beforeEach(() => {
+		jest.spyOn(InteropService.instance(), 'export').mockResolvedValue({ warnings: [] });
+		jest.spyOn(shim, 'setTimeout').mockImplementation(callback => void callback());
+		mockWindow.webContents.on.mockImplementation((_event, callback) => callback());
+		mockWindow.webContents.executeJavaScript.mockImplementation(script => Promise.resolve(window.eval(script)));
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)